modules/combined/tests/contact_verification/patch_tests/plane_3/plane3_template1.i

```diff
@@ -269,8 +269,8 @@
   #Preconditioned JFNK (default)
   solve_type = 'PJFNK'
 
-  petsc_options_iname = '-pc_type -pc_factor_mat_solver_package'
-  petsc_options_value = 'lu     superlu_dist'
+  petsc_options_iname = '-pc_type -pc_factor_mat_solver_package -mat_superlu_dist_fact'
+  petsc_options_value = 'lu            superlu_dist               SamePattern'
```

Comment:
There are three example rods in BISON, but more tests in the assessments may be helped by this option